app/database_etl/owid_ingestion_handler/owid_data_handler.py

```diff
@@ -0,0 +1,84 @@
+import pandas as pd
+import datetime
+import os
+
+from app.database_etl.location_utils import get_alternative_names, get_country_code
+
+CSV_DIR = os.getenv('CSV_DIR')
+
+vaccination_df = pd.read_csv(CSV_DIR + "vaccinations.csv")[['date', 'iso_code', 'people_vaccinated_per_hundred', 'people_fully_vaccinated_per_hundred']]
```

Comment:
Same thing here - not sure why these need to be global constants. Each of these is only used in one of the functions below. Let's just define these variables in the functions they're used. Makes it harder to understand where these variables are used if they're defined at the top outside the functions

Reply:
Again – these functions are called hundreds of times. For each function call, we are going to be using system resources to read the CSV, load it into memory, then have Python garbage collect when the function returns.

I understand that it may difficult to understand where the variables come from. Is there another solution that would allow the code to be clearer, while not eating up resources?

Reply:
@atatmaja thoughts?

Reply:
The only other solution I can think of is defining these DFs in `create_dashboard_source_df` and passing them into the owid_handler functions as params. Tbh I think that's more unclear than this global constant approach so I'm okay with this for now @simonarocco @abeljoseph